src/bluehawk/parser/visitor/makeCstVisitor.ts

```diff
@@ -305,17 +311,39 @@ export function makeCstVisitor(
             )
           );
         }
-
-        assert(context.Newline[0].endLine !== undefined);
-        assert(context.Newline[0].endOffset !== undefined);
+        assert(context.CommandStart[0].endOffset !== undefined);
+        assert(context.CommandStart[0].endLine !== undefined);
         assert(context.CommandEnd[0].startColumn !== undefined);
         assert(context.CommandEnd[0].startLine !== undefined);
-        if (context.chunk != undefined) {
+
+        let lineStart: number;
+        let offsetStart: number;
+        if (
+          context.Newline !== undefined &&
+          context.Newline[0] !== undefined &&
+          context.Newline[0].endLine !== undefined &&
+          context.Newline[0].endOffset !== undefined
+        ) {
+          lineStart = context.Newline[0].endLine + 1;
+          offsetStart = context.Newline[0].endOffset + 1;
+        } else {
+          // start and end offsets for block commands with uncommented contents
+          assert(context.blockCommandUncommentedContents !== undefined);
+          let NewLineTokenArr = context.blockCommandUncommentedContents[0]
```

Comment:
Prefer const. Your vscode should be set up to automatically fix these upon save (part of eslint/prettier)
```suggestion
          const NewLineTokenArr = context.blockCommandUncommentedContents[0]
```

Reply:
Identical response to [here](https://github.com/mongodb-university/Bluehawk/pull/68#discussion_r704759504). Made the change, but would be curious to hear your thoughts around Kyle Simpson's argument against this.

Reply:
I don't know who Kyle Simpson is.

Reply:
(I actually do, but I'm making the point that if we're going to appeal to authority, I'd rather it be tool-supported like our existing (and minimal!) eslint/prettier configuration)